app/views/order_notifier/customer.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1>Take-Away Order Receipt</h1>
+
+<p>
+  Hello <%= @order.user.name -%>,
+  <br><br>
+  Your order for take-away food was received.
+  <br><br>
+  It will be ready by your pick-up time of <%=@order.pickup_time -%>
+  <br><br>
+  You ordered these <%=@order.order_items.length -%> items:
```

Comment:
@patmbolger what do you think about having this `You ordered the following item(s)`
```ruby
You ordered the following <%= "item".pluralize(@order.order_items.length) %>
```